cn.lpkt.serverbox.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=ServerBox
+Exec=run.sh
+Icon=cn.lpkt.serverbox.png
```

Comment:
still not solved https://github.com/flathub/flathub/pull/6117#discussion_r1939376649